src/main/resources/application.properties

```diff
@@ -0,0 +1,17 @@
+server.port=8080
+
+#Database
+spring.datasource.url=jdbc:postgresql://localhost:5432/tournament
+spring.datasource.username=postgres
+spring.datasource.password=root
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.jpa.database=postgresql
+spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
+
+#Hibernate
+spring.jpa.hibernate.ddl-auto=none
+spring.jpa.show-sql=true
+spring.jpa.properties.hibernate.format_sql=true
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQL81Dialect
+spring.jpa.properties.hibernate.temp.use_jdbc_metadata_defaults = false
```

Comment:
I do not think you need it.
This just make things more complicated that it is needed.